cmd/argo/commands/submit.go

```diff
@@ -248,3 +271,100 @@ func waitOrWatch(workflowNames []string, cliSubmitOpts cliSubmitOpts) {
 		watchWorkflow(workflowNames[0])
 	}
 }
+
+func parseParameters(opts *util.SubmitOpts) ([]wfv1.Parameter, error) {
```

Comment:
This function is copy/pasted from `util.go`. We should extract that function so it can be used in multiple places instead of simply copy/pasting them. One function is way easier to maintain than two.